### PR TITLE
Get submission sheet test

### PR DESCRIPTION
**Pull request by karthikmsqa**
# Description

This PR includes the following proposed change(s):

- Get submission sheet endpoint test

## Type of change

- [x] New test(non-breaking change)

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

- Tested locally and Github actions

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/getSubmissionSheetSD.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    @Given("user account is authenticated")
+    public void userAccountIsAuthenticated() {
+
+        actualUserIdentity = oauthService.getUserIdentity();
+    }
+
+    @When("user submits request to get submission sheet")
+    public void getSubmissionSheetRequest() {
+        logger.info("Requesting submission sheet from filing packages");
```
Comment (KDNeufeld) [inline tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/getSubmissionSheetSD.java:38]: You may want to flip these log statements to `.debug` or `.trace`, leaving `.info` for high-level course-grained information (ie. app started up with these parameters).

Comment (karthikmsqa) [reply tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/getSubmissionSheetSD.java:38]: Good point! I wasn't sure if .debug will be too much information to parse through or will be take a bit extra time on the CI test runs when implemented for all requests. But a worth trying... Thanks @KDNeufeld!

Comment (alexjoybc) [reply tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/getSubmissionSheetSD.java:38]: I don't see too much issue having the integration test verbose, something we could consider, but I think we always wants to see those logs in the output, what do you guys think ?

**Review by alexjoybc — APPROVED** [review]
Looks good @karthikmsqa

**Pull request merged**